tasks/prepare-binaries.js

```diff
@@ -34,7 +34,7 @@ async function createPackage (platforms) {
 async function getBrowsers (platforms) {
   await Promise.all(Object.entries(platforms).map(async ([name, platform], index) => {
     const puppeteerPlatform = platform.puppeteerPlatform || name
-    await puppeteer
+    return puppeteer
```

Comment:
no need for `await` here, cause eventually all promises are awaited in by `await Promise.all()`.